drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -166,7 +166,10 @@ def items_without_answers():
             items = copy.deepcopy(self.data.get('items', ''))
             for item in items:
                 del item['feedback']
-                del item['zone']
+                # Switching to `pop` from `del`, as we no longer have any guarantee of
+                # either `zone` or `zones` being present.
```

Comment:
@haikuginger Thanks for adding this comment!

Right now it reads a bit like a commit message. How about something like:

> Use `pop` instead of `del` to remove target zone info. This is necessary because we don't have any guarantee of either `zone` or `zones` being present (legacy instances of this block allow only a single `zone` per item; new instances allow authors to specify a list of `zones`).